UNITTESTS/CMakeLists.txt

```diff
@@ -1,66 +1,35 @@
-cmake_minimum_required(VERSION 3.0.2)
+# Copyright (c) 2021 ARM Limited. All rights reserved.
```

Comment:
indentation in this file is a bit off in places, maybe worth just going through and cleaning it up